src/Controller/DataController.java

```diff
@@ -129,6 +117,8 @@ static private HubFile processUpdateHubFile(NodeList nList) throws Exception
                             case "examEvent":
                                 // not yet added
                                 break;
+                        default:
```

Comment:
What is the purpose of adding this default clause?  Should it do something (like throw an exception or log an error)?